src/main/java/org/elasql/perf/tpart/workload/TransactionFeatures.java

```diff
@@ -16,8 +18,37 @@
 	// Defines a read-only list for feature keys
 	public static final List<String> FEATURE_KEYS;
 	
+	private static class FeatureKeys {
+		private List<String> featureKeys = new ArrayList<String>();
+		private int serverCount; 
+		
+		public FeatureKeys(int serverCount) {
+			this.serverCount = serverCount;
+		}
+		
+		public boolean add(String key) {
+			return featureKeys.add(key);
+		}
+		
+		public void addWithServerCount(String key) {
+			for (int serverId = 0; serverId < serverCount; serverId++) {
+				String keyWithServerId = getKeyWithServerId(key, serverId);
+				featureKeys.add(keyWithServerId);
+			}
+		}
+		
+		public List<String> getList() {
+			return featureKeys;
+		}
+	}
+	
+	public static String getKeyWithServerId(String key, int serverId) {
+		// %-3d means the field width is 3 and it is left justification
+		return String.format("%s - Server %-3d", key, serverId);
```

Comment:
Is there any reason to format the number using `%-3d`? Why not just `%d`?

Reply:
I would just like to make the length of column name consistent.
`%d` would be OK as well. I'll revise it.